keyboards/keebio/bfo9000/keymaps/shadyproject/config.h

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
This file should be removed, since it's not setting anything.

Reply:
removed in latest pull request